### ISSUE TITLE
Quality settings: keep better quality

### ISSUE THREAD

**Issue by Audistic** [pymedusa/Medusa#6060]
In general I'm perfectly happy watching new episodes in SDTV quality. For some series however I eventually want to replace the initial file with 720/1080 hd season packs, which I download manually.

With the current way of setting quality requirements i'm running into issues:
when setting prefered/allowed to SD, medusa will replace manually downloaded HD seasonpacks, with SDTV.
when setting prefered/allowed to SD/HD, medusa will automatically download SD/HD.
Ofcourse, this is expected behaviour.
However I'm looking for an option, to override prefered quality when better quality is allready downloaded.

**Comment by Yellow1144** [comment]
Yes, This would be great. I have recently had an experience where I had a show in 720 (the default quality) and I manually downloaded a 1080 bluray season pack and it post-processed properly. The next day I noticed that overnight Medusa had downloaded and replaced my bluray quality with 720HDTV again. Was a bit maddening. Re-downloaded the Bluray and set season to archived. 

I thought that Medusa would check to see that the episodes already in the library were at a higher quality than the default, and, finding a higher quality present, leave them as is. Is there a particular reason for this behavior or can it be fixed to skip the download if episodes already exist at a higher quality?

**Comment by mtclemente** [comment]
I would like to put my support behind this as well.  I've experienced this and it's very frustrating when I download a 1080p episode and Medusa overwrites it with a 720p because that is the "preferred".  I've only been able to get around this by setting certain shows that I want 1080p for to preferred 720/1080p.  But would love it if Medusa would recognize for a 720p show that has 1080p episodes that the higher quality should supersede.
